python_modules/libraries/dagster-dg/dagster_dg_tests/utils_tests/test_sample_yaml.py

```diff
@@ -26,8 +26,8 @@ def test_generate_sample_yaml():
 
 params:
   sub_scoped: # Available scope: {'outer_scope'}
-    str_field: '...'
-    int_field: 0
+    str_field: '...' # Available scope: {'outer_scope'}
```

Comment:
it was actually a bug that these comments weren't showing up before

in terms of desired behavior, the previous output might actually have been nicer but now it's much easier to choose which way we want this to appear